src/Sulu/Bundle/MediaBundle/Tests/Functional/Controller/MediaStreamControllerTest.php

```diff
@@ -121,6 +121,31 @@ public function testGetImageAction()
         $this->assertSame('image/gif', $client->getResponse()->headers->get('Content-Type'));
     }
 
+    public function testGetImageActionSvg()
+    {
+        if (!class_exists(\Imagick::class)) {
```

Comment:
Don't understand this skip. If imagick is not available it should use gd and in this case it should not matter as it should use the svg adapter.

Reply:
had following error: https://travis-ci.org/sulu/sulu/jobs/633654526#L1012

i think gd is not able to to render svg as png (last check in this testcase).

will change that to only check the conversion when imagick is available